### PR TITLE
Marketing review updates:

### PR DESCRIPTION
**Pull request by miakobchuk**
## Purpose of this pull request

Marketing review updates:
Information added to guidelines about submission fields that commonly cause rejections in EQP

## Affected DevDocs pages

https://devdocs.magento.com/marketplace/sellers/marketing-review-guidelines.html

### CODE REVIEW

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:13]:
```suggestion
*  The product name must indicate the purpose of the extension or theme.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:14]:
```suggestion
*  The product description must be well-written and provide an overview of the features and benefits.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:21]:
```suggestion
*  Choose a title that is descriptive and concise. The extension title can be up to five words in length and must be in English.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:23]:
```suggestion
   *  Do not use _Magento_ or _M2_.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:24]: I would suggest in this and the following bulleted items that you either use italics or quotations (I suggest italics) and not both.

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:25]:
```suggestion
   *  The words, _Connector_ and _Integration_ can be used if applicable.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:26]:
```suggestion
*  Use _&_ instead of _And_, such as in _Order & Save_ (not _Order and Save_).
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:27]:
```suggestion
* Capitalize the first letter of each word, such as _Mass Order Export_ (not _Mass order export_).
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:28]:
```suggestion
   *  Do not use all capital letters.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:29]:
```suggestion
*  Do not use punctuation unless it’s part of the actual company name. (_"InstantSearch+"_, _"Yahoo!"_)
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:31]:
```suggestion
*  Do not add underscores between words in the package name, such as (_Extension_Title_).
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:32]:
```suggestion
*  Do not use the Developer Name in the title, such as _Print Order PDF_ (not _Fooman Print Order PDF_).
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:33]:
```suggestion
   *  If the developer is the same name as the service being integrated, you may use the develop name in the title. For instance, if Visa submits an integration, they should include _Visa_ since it’s the name of the service.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:34]:
```suggestion
*  You can add the Magento version, such as 2.3, if two separate extensions must be on the store.
```

Comment (shrielenee) [reply src/marketplace/sellers/marketing-review-guidelines.md:34]: On the store? Or in the store? Check phrasing.

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [reply src/marketplace/sellers/marketing-review-guidelines.md:24]: Please check all bulleted items for formatting and complete sentences.

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:41]:
```suggestion
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:40]:
```suggestion
*  The extension icon must reflect the extension with words and/or images. Text is not required, but an extension title can be added to the icon.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:42]:
```suggestion
   *  If the extension title is added to the icon, it should exactly match the extension title.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:44]:
```suggestion
*  Do not use _Magento_ or _M2_.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:45]:
```suggestion
*  Do not use the Magento Logo.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:46]:
```suggestion
*  Do not use the developer logo instead of the extension icon.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:48]:
```suggestion
*  Limit the use of the word _Pro_.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:49]:
```suggestion
*  Less is more. Avoid minute detail that is hard to understand.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:50]:
```suggestion
*  For 1st party companies, we recommend using the same image you use on Twitter, LinkedIn, etc.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:56]:
```suggestion
## User guide, reference manual, or installation guide
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:60]:
```suggestion
*  Must cover all features of the extension.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:61]:
```suggestion
*  Must not direct users to make purchases on sites other than Marketplace.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:62]:
```suggestion
*  If an extension title is added into the documentation, it must not include _Magento_. Use _For Magento_ after the title if needed.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:65]:
```suggestion
*  At least one piece of uploaded documentation must be in English.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:72]:
```suggestion
*  Items listed on thr Magento Marketplace can be offered for free or sold as commercial products.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:73]:
```suggestion
*  The price of items offered for sale on the Magento Marketplace can range anywhere from $25.00 to $999,999.00, and must reflect fair market value.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:75]:
```suggestion
*  Items sold on the Magento Marketplace are subject to a revenue-sharing fee that is paid by the developer.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:76]:
```suggestion
*  Any additional fees or subscription pricing—including fees charged by a third-party company for integrated services—must be included in the opening paragraph of the Long Description with bold formatting.
```

Comment (shrielenee) [reply src/marketplace/sellers/marketing-review-guidelines.md:76]: You use "third-party" here but "1st party" higher up in the doc. Pick one convention and use the same for both instances.

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:77]:
```suggestion
*  Magento recommends $99 for Open Source and $199 for Commerce extension pricing.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:78]:
```suggestion
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:79]:
```suggestion
```

**Review by shrielenee — CHANGES_REQUESTED** [review]
Lot of small changes requested. Let em know if you need assistance with the changes or have questions on how to implement them. Thanks!

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:24]:
```suggestion
*  Do not include the words _Extension_, _Module_ or _Plugin_.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:29]:
```suggestion
*  Do not use punctuation unless it’s part of the actual company name, such as _InstantSearch+_ or _Yahoo!_.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:30]:
```suggestion
   *  _/_ and _-_ are acceptable in certain circumstances, such as for _Two-Factor_ or _Import / Export_.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [inline src/marketplace/sellers/marketing-review-guidelines.md:49]:
```suggestion
*  For first-party companies, we recommend using the same image you use on Twitter, LinkedIn, etc.
```

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (shrielenee) [reply src/marketplace/sellers/marketing-review-guidelines.md:24]: You didn't apply changes per my comment above, so I've done the work for you and added the suggestions for you to commit.

I will be requesting a review from @dobooth as well before merging.

**Review by shrielenee — APPROVED** [review]
Approved with suggested changes.

**Comment by dobooth** [comment]
Hi @miakobchuk If you enable maintainer edit-ability on the fork, the devdocs team can make changes. https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by miakobchuk** [comment]
Oh. i have not noticed that it was not merged because one space on line 27
27:1 MD030/list-marker-space Spaces after list markers [Expected: 2; Actual: 1]
Can it be fixed in GitHub UI?

**Comment by dobooth** [comment]
@miakobchuk It should be easy to fix in the Github UI. Just edit the file from the Files tab and add the space. But you will need to update the branch as well.

File: src/marketplace/sellers/marketing-review-guidelines.md
Comment (dobooth) [inline src/marketplace/sellers/marketing-review-guidelines.md:71]:
```suggestion
*  Items listed on the Magento Marketplace can be offered for free or sold as commercial products.
```

**Review by dobooth — CHANGES_REQUESTED** [review]

**Comment by dobooth** [comment]
@miakobchuk I found one typo that needs fixing. Once you have made the changes, ping me on Slack. I will need you to update the branch and then I can merge this.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**
